diff -Nru wmakerconf-2.3/data/po/Makefile.am wmakerconf-2.3.new/data/po/Makefile.am
--- wmakerconf-2.3/data/po/Makefile.am	Thu Oct  7 06:58:19 1999
+++ wmakerconf-2.3.new/data/po/Makefile.am	Thu Oct  7 06:57:58 1999
@@ -66,7 +66,7 @@
 	  esac; \
 	  echo $$destdir; \
 	  lang=`echo $$cat | sed 's/\$(CATOBJEXT)$$//'`; \
-	  dir=$$destdir/$$lang/LC_MESSAGES; \
+	  dir=$(DESTDIR)$$destdir/$$lang/LC_MESSAGES; \
 	  $(mkinstalldirs) $$dir; \
 	  if test -r $$cat; then \
 	    $(INSTALL_DATA) $$cat $$dir/$(PACKAGE)$(INSTOBJEXT); \
diff -ur wmakerconf-2.8.1/Makefile.am wmakerconf-2.8.1.patched/Makefile.am
--- wmakerconf-2.8.1/Makefile.am	Fri Mar 16 18:19:23 2001
+++ wmakerconf-2.8.1.patched/Makefile.am	Fri Mar 23 23:06:50 2001
@@ -22,9 +22,9 @@
 GNOMEAPP = @GNOMEAPP@
 
 install-data-local: $(srcdir)/wmakerconf.desktop
-	$(mkinstalldirs) $(GNOMEDIR)
+	$(mkinstalldirs) $(DESTDIR)$(GNOMEDIR)
 	@list='$(GNOMEAPP)'; for p in $$list; do \
 		echo "$(INSTALL_DATA) $(srcdir)/$$p $(GNOMEDIR)/$$p"; \
-		$(INSTALL_DATA) $(srcdir)/$$p $(GNOMEDIR)/$$p; \
+		$(INSTALL_DATA) $(srcdir)/$$p $(DESTDIR)$(GNOMEDIR)/$$p; \
 	done
 
