--- wmakerconf-2.5/data/po/Makefile.am.pius	Wed Dec 29 18:31:44 1999
+++ wmakerconf-2.5/data/po/Makefile.am	Tue Jan 25 15:13:07 2000
@@ -7,8 +7,8 @@
 #EXTRA_DIST = ca.po cs.po da.po de.po el.po es.po fi.po fr.po hu.po it.po \
 #	     ja.po ko.po no.po pl.po pt_BR.po ro.po ru.po sv.po tr.po $(CATALOGS)
 
-CATALOGS   = de.gmo ja.gmo
-EXTRA_DIST = de.po ja.po $(CATALOGS)
+CATALOGS   = de.gmo ja.gmo pl.gmo
+EXTRA_DIST = de.po ja.po pl.po $(CATALOGS)
 
 CATOBJEXT  = .gmo
 INSTOBJEXT = .mo
@@ -101,4 +101,4 @@
 	for file in $$dists; do \
 	  ln $(srcdir)/$$file $(distdir) 2> /dev/null \
 	    || cp -p $(srcdir)/$$file $(distdir); \
-	done
\ No newline at end of file
+	done
